### ISSUE TITLE
bug: [undefined: atomic.Bool ]

### ISSUE THREAD

**Issue by Program-repo** [lrstanley/bubblezone#9]
### 🌧 Describe the problem

When I attend to run the full-lipgloss example, I got the following error message:
# github.com/lrstanley/bubblezone
../go/pkg/mod/github.com/lrstanley/bubblezone@v0.0.0-20221015234447-8db0faca329d/manager.go:54:17: undefined: atomic.Bool

### ⛅ Expected behavior

I expect to run the full test

### 🔄 Minimal reproduction

1 in my directory full-lipgloss I have the following files:
dialog.go  go.mod  go.sum  history.go  list.go  main.go  tabs.go  util.go
2 Then I run go run .     and I got the error message

### 💠 Version: bubblezone

github.com/lrstanley/bubblezone v0.0.0-20221015234447-8db0faca329d h1:cYNT20vgHPsn7fK6cx/zy3iCaPnlcztWDerhx4iPTZ8= github.com/lrstanley/bubblezone v0.0.0-20221015234447-8db0faca329d/go.mod h1:u+P0PRRatV4IhcL0LncRZAew9nlYvTQvgYbEu5h1iw0= g

### 🖥 Version: Operating system

linux/other

### ⚙ Additional context

I am running linux debian 11 in a Pixelbook .
go version is go1.18.7 linux/amd64

### 🤝 Requirements

- [X] I believe the problem I'm facing is a bug, and is not intended behavior. [Post here if you're not sure](../discussions/new?category=q-a).
- [X] I have confirmed that someone else has not [submitted a similar bug report](../labels/bug).

**Comment by lrstanley** [comment]
This is due to `atomic.Bool` being added in Go 1.19. I'll bump the required Go version in the `go.mod` file as well to make it more clear that Go 1.19 will be required moving forward.

If you are unable to update your Go version, the last version that supports Go 1.18 can be pulled via this command (was just a week or so ago that I added the new 1.19 requirement):

```console
$ go get github.com/lrstanley/bubblezone@fc2e2a5f029495535fe32183ec6110b2aba2692b
```

**Comment by lrstanley** [comment]
Pinning to a minimum of Go 1.19 added in https://github.com/lrstanley/bubblezone/commit/b3469cc5a65953ac7af1d38bf6ba5391eba93751.

**Comment by Program-repo** [comment]
Most appreciated Liam!On Oct 29, 2022 7:33 p.m., Liam Stanley ***@***.***> wrote:
Closed #9 as completed.
